### DISCUSSION TITLE
when run "sbt compile", then occur a error: object Driver is not a member of package firrtl

### DISCUSSION THREAD

**Discussion by athurwo** [chipsalliance/chisel#2266] [General]
this code is line 54 in chisel3/src/main/scala/chisel3/Driver.scala,

`firrtl.Driver.execute(optionsManager) match {
      case _: FirrtlExecutionSuccess => true
      case _: FirrtlExecutionFailure => false
    }`

I have check the Driver in firrtl API, it had deprecated since version FIRRTL 1.2[https://www.chisel-lang.org/api/firrtl/latest/firrtl/Driver$.html](url)

what should I do to resolve this error ？
thanks.

**Comment by jackkoenig** [comment]
Sorry for my delayed response. Chisel3 master was broken for a while due to removed APIs in firrtl, but we have since rectified that breakage and it should be working now. Thus to resolve the error, please update to latest master.
